### ISSUE TITLE
Integration test cases with chocolatey

### ISSUE THREAD

**Issue by anki2189** [strongbox/strongbox#1714]
# Task Description

Strongbox has Nuget support which is why we are indirectly supporting `choco`. 
We need to add some integration tests to [strongbox/strongbox-web-integration-tests](https://github.com/strongbox/strongbox-web-integration-tests) which cover all of the commands you would need to use in order to work with `choco`.

# Tasks

The following tasks will need to be carried out:
* [ ] [Read the documentation](https://strongbox.github.io/user-guide/tool-integration/nuget-chocolatey-example.html)
* [ ] Write integration test for the commands:
    * [ ] [Save API key in choco](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#save-api-key-in-choco)
    * [ ] [Add Strongbox repository to Chocolatey package sources](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#add-strongbox-repository-to-chocolatey-package-sources)
    * [ ] [Create a package](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#create-a-package)
    * [ ] [Make chocolatey Nuget package](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#make-chocolatey-nuget-package)
    * [ ] [Push Nuget package into Strongbox repository](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#push-nuget-package-into-strongbox-repository)
    * [ ] [Search for packages in Strongbox repositories](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#search-for-packages-in-strongbox-repositories)
    * [ ] [Install package using Strongbox source](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#install-a-package)
    * [ ] [Delete package](https://deploy-preview-78--strongbox.netlify.com/user-guide/tool-integration/nuget-chocolatey-example.html#uninstall-a-package)
* [ ] Update landing doc page to include `Choco` (after Nuget V2) in the [Layout Providers](https://strongbox.github.io/index.html) section.

# Task Relationships

This task:
* Depends on: #1709 
* Relates to: #1685 #1113, strongbox/strongbox-docs#78

# Useful Links

* [s-w-i-t with nuget](https://github.com/strongbox/strongbox-web-integration-tests/tree/master/nuget)
* [how to write tests](https://strongbox.github.io/developer-guide/writing-web-integration-tests.html)
* [Chocolatey commands](https://chocolatey.org/docs/commands-reference)
* [hello-choco-example](https://github.com/strongbox/strongbox-examples/pull/15)

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @fuss86
  * @sbespalov
  * @steve-todorov

**Comment by steve-todorov** [comment]
This is temporarily on hold while I fix some issues around strongbox/ci-build-images#8 hopefully within the next few hours. @TheCakeIsNaOH would you be interested in helping out with writing the integration test case and finalizing strongbox/strongbox-examples#15 (if there is anything more to be done there) ? Sorry for the long delay, but we were simply blocked by chocolatey. :)

**Comment by anki2189** [comment]
May I pick this.?

**Comment by carlspring** [comment]
Please, proceed!
